BZLMOD_SUPPORT.md

```diff
@@ -35,3 +35,22 @@ This rule set does not have full feature partity with the older `WORKSPACE` type
 2. Gazelle does not support finding deps in sub-modules.  For instance we can have a dep like ` "@our_other_module//other_module/pkg:lib",` in a `py_test` definition.
 
 Check ["issues"](/bazelbuild/rules_python/issues) for an up to date list.
+
+## Differences in behavior from WORKSPACE
+
+### Default toolchain is not the local system Python
+
+Under bzlmod, the default toolchain is no longer based on the locally installed
+system toolchain. Instead, a recent Python version using the pre-built,
+standalone runtimes are used.
+
+If you need to use the local system python, then it's suggested that you
+setup and configure your own toolchain and register it. Note that using the
+local system's Python is not advised because will vary between users and
```

Comment:
So it sounds like we don’t need a local python installed, but we still do. Please add a note that we need Python installed locally to shim the use of the hermetic toolchain.

Reply:
Well, OK, I added a note about that. However, this section is just about the toolchain, which is separate from the bootstrapping process. Regardless of what toolchain you use or how (local, hermetic, your own, etc), the bootstrap is the same: a local python is necessary to do it.

I'd like to reference the issue about this, but I can't find it. If you know it, LMK.